rero_ebooks/ext.py

```diff
@@ -54,4 +50,3 @@ def init_config(self, app):
     @staticmethod
     def register_signals(app):
         """Register Zenodo Deposit signals."""
```

Comment:
?? zenodo??

Reply:
Old code, I have changed to: `Register Ebooks signals.`